deps/exokit-bindings/webglcontext/src/webgl.cc

```diff
@@ -930,6 +935,21 @@ ColorMaskState &ColorMaskState::operator=(const ColorMaskState &colorMaskState)
 std::pair<Local<Object>, Local<FunctionTemplate>> WebGLRenderingContext::Initialize(Isolate *isolate) {
   // Nan::EscapableHandleScope scope;
 
+#if defined(ANDROID) || defined(LUMIN)
```

Comment:
Is this the right place to do the initialization? I think it will happen once for every `window`. We should endeavor to do this initialization only once, globally.

Reply:
5bf4b3f